lametro/events.py

```diff
@@ -110,16 +110,20 @@ def api_events(self, *args, **kwargs):
             # should have a pair.
 
             if partial_scrape:
-                partner_event = self._find_partner(unpaired_event)
+                try:
+                    partner_event = self._find_partner(unpaired_event)
+
+                    spanish_start_date = datetime.datetime(2018, 5, 15, 0, 0, 0, 0)
+                    event_date = datetime.datetime.strptime(unpaired_event['EventDate'], '%Y-%m-%dT%H:%M:%S')
 
-                spanish_start_date = datetime.datetime(2018, 5, 15, 0, 0, 0, 0)
-                event_date = datetime.datetime.strptime(unpaired_event['EventDate'], '%Y-%m-%dT%H:%M:%S')
+                    if partner_event is not None:
+                        yield partner_event
 
-                if partner_event is not None:
-                    yield partner_event
+                    elif event_date > spanish_start_date and unpaired_event.is_spanish:
+                        raise UnmatchedEventError(unpaired_event)
```

Comment:
i think it's a little funny to manually raise an error just to immediately catch it. 

it seems like maybe what we want to do is raise a Warning instead? 

https://docs.python.org/3/library/warnings.html

Reply:
I agree. Will we just not raise `UnmatchedEventError`s at all going forward? Should we keep the [error definition](https://github.com/opencivicdata/scrapers-us-municipal/blob/master/lametro/events.py#L11-L26)?